lib/nrf_modem_lib/shell/Kconfig

```diff
@@ -0,0 +1,5 @@
+config NRF_MODEM_LIB_TRACE_SHELL
+	bool "Shell commands for modem traces"
+	default y
```

Comment:
It's only being enabled by default if the dependencies are being met. So `...if SHELL && NRF_MODEM_LIB_TRACE` doesn't have any effect.

Reply:
Ok, I was thinking this would give an error because of unsatisfied dependencies, but I see now that it is only shown if this is enabled by the user when the dependencies are not set.